### DISCUSSION TITLE
When comes 0.2.6 ?

### DISCUSSION THREAD

**Discussion by karesake** [samemory/homebridge-eufy-security-control#76] [Q&A]
Hey guys,

Can tell me anyone, when comes the new 0.2.6 version as a npm update out ? 

I cant install the dev version. It gives me out an error

**Comment by njuart** [comment]
I've managed to install dev, but it also gives same errors in logs as before and causes HB reboot loop. Even though my issue was closed as resolved https://github.com/samemory/homebridge-eufy-security/issues/56

**Reply by njuart** [reply]
Guys, maybe you can give us just a small hint when we can expect new version? =) Thx in advance

**Reply by schliemann** [reply]
Maybe @samemory can give an estimate.

**Reply by samemory** [reply]
Still having issues with the dependencies. Hoping to find a fix soon!

**Reply by njuart** [reply]
> Still having issues with the dependencies. Hoping to find a fix soon!

Thx for the efforts, ready to check as soon as it's done =)
